### ISSUE TITLE
Doesn't work on Opera GX core: 93.0.4585.84

### ISSUE THREAD

**Issue by usersmthuser** [limjeck/osuplus#77]
Browser identification
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36 OPR/93.0.0.0
Attaching logs
[osu.ppy.sh-1674679628150.log](https://github.com/limjeck/osuplus/files/10503565/osu.ppy.sh-1674679628150.log)

**Comment by limjeck** [comment]
Yeah some Opera GX users seem to have issues, but some say it works fine. It works fine for me, make sure you are using the Tampermonkey from https://addons.opera.com/en/extensions/details/tampermonkey-beta/. Try and see if ANY userscript works for you.

**Comment by usersmthuser** [comment]
Im using same tampermonkey, but I reinstalled it just in case, and it still not working. Other scripts working fine, but osuplus is... just not working idk why, it's also might be an osu website issue, cuz Im getting errors on beatmaps scripts
